web/client/src/components/EventsMap/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react'
+import mapboxgl from 'mapbox-gl'
+import { Dropdown, MultiSelect } from 'carbon-components-react'
+
+const items = [
+  { id: 'timeframe-past-7', text: 'Past 7 Days' },
+  { id: 'timeframe-past-30', text: 'Past 30 Days' },
+  { id: 'timeframe-past-90', text: 'Past 90 Days' },
+  { id: 'timeframe-past-year', text: 'Past Year' },
+]
+
+console.log(process.env.REACT_APP_MAPBOX_ACCESS_TOKEN)
+
+mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_ACCESS_TOKEN
+
+export default class EventsMap extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      lng: -99.3533,
+      lat: 20.8857,
+      zoom: 4.65,
+    }
+  }
+
+  componentDidMount() {
+    const map = new mapboxgl.Map({
+      container: this.mapWrapper,
+      style: 'mapbox://styles/mapbox/dark-v10',
+      center: [this.state.lng, this.state.lat],
+      zoom: this.state.zoom,
+      attributionControl: false,
+    }).addControl(
+      new mapboxgl.AttributionControl({
+        compact: true,
+      })
+    )
+
+    map.on('move', () => {
```

Comment:
Inconsistent use of `function() {}` and `() => {}`
on line 39 and 55

Reply:
Well, honestly, to conform the standard we talked about, the entire component needs to be reworked into a functional, arrow component -- its the last surviving class-based component from my overhaul. The thing is, this component is going to go through a few revisions in the next week or so since we're refining the design, so I'd rather save refactoring that until later, and get these changes out to everyone that is looking to work on the app.